content/learn/08.contributions/04.arduino-creating-content-guide/arduino-creating-content-guide.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Writing Content for Arduino'
+description: 'How to write content for the Arduino Documentation Website'
+tags: 
+  - Styleguide
+  - Guidelines
+---
+
+## Adding your own content
+
+You are welcome to add your own content to the Arduino Documentation repository. You can contribute with the following content:
+
+|Content|Description|Example|
+|-------|-----------|-------|
+|Tutorial|Learn how to do something.|[Control Built-in RGB LED over Wi-Fi with Nano RP2040 Connect](https://docs.arduino.cc/tutorials/nano-rp2040-connect/rp2040-web-server-rgb)|
+|Article|Learn about a specific topic.|[Multimeter Basics](https://docs.arduino.cc/learn/electronics/multimeter-basics)|
+|Nugget|Smaller tutorial with less information and more straight to the example.|[Analog Read Serial](https://docs.arduino.cc/built-in-examples/basics/AnalogReadSerial)|
+|Project|Learn how to build something.|[DIY Photoshop Editing Console using Arduino Nano RP2040 Connect](https://create.arduino.cc/projecthub/jithinsanal1610/diy-photoshop-editing-console-using-arduino-nano-rp-2040-a43e97?ref=search&ref_id=nano%20rp2040%20connect&offset=0)|
+
+\*Note that the contributed Projects should be added to **Project Hub** instead of on the Arduino Documentation website.
+
+***You can also contribute to our content by creating your own libraries. Read more about that [here](/learn/contributions/arduino-library-style-guide).***
+
+
+For us to be able to approve your contribution, you should [follow the guidelines](/learn/contributions/arduino-writing-style-guide) on how to structure and write your content.
+
+
+## Styleguides
+
+If you want to write your own content, please read the following guides on how to write in a user friendly way.
+
+* [Arduino Style Guide for Writing Content](/learn/contributions/arduino-writing-style-guide)
+
+
+If you want to contribute with your own content to the Arduino Documentation website, please read the following documentation on how to do just that.
+
+* [Contribution Content](https://github.com/arduino/docs-content/contribution-templates)
```

Comment:
```suggestion
* [Contribution Content](/docs-content/contribution-templates)
```

Reply:
@karlsoderby not sure about this link. Is it correct?

Reply:
Here is the correct link: https://github.com/arduino/docs-content/tree/main/contribution-templates